include/SDL3/SDL_gpu.h

```diff
@@ -420,13 +420,42 @@ typedef struct SDL_GpuViewport
     float maxDepth;
 } SDL_GpuViewport;
 
+typedef struct SDL_GpuTransferBufferImage
+{
+    SDL_GpuTransferBuffer *transferBuffer;
+    Uint32 offset;      /* starting location of the image data */
+    Uint32 imagePitch;  /* number of pixels from one row to the next */
+    Uint32 imageHeight; /* number of rows from one layer/depth-slice to the next */
+} SDL_GpuTransferBufferImage;
```

Comment:
TransferBufferImage is kind of a confusing name, maybe ImageTransfer would be better

Reply:
Copying some ideas from Discord: TransferBufferSurfaceInfo, TextureTransferInfo - main thing is Info since this is basically TransferBufferLocation with image layout info.

Reply:
TextureTransferInfo sounds the best to me

Reply:
Went with TextureTransferInfo